### PR TITLE
Fix facade-code-coverage.bat for System.Private.CoreLib changes

### PR DESCRIPTION
**Pull request by hughbe**
Ran locally, script works as expected

/cc @ianhays @stpehentoub

### CODE REVIEW

File: Documentation/building/code-coverage.md [hunk 1]
```diff
@@ -70,12 +70,13 @@ The following steps can be used manually to produce a coverage report, but a cus
 1. Build the local test project (`msbuild /T:Build`)
 3. Build coreclr locally in Debug or Release (`build.cmd all Debug skiptests`)
 2. Navigate to the built test directory in the corefx bin (e.g. `bin/tests/AnyOS.AnyCPU.Debug/System.Runtime/netcoreapp1.0` for `System.Runtime`
-4. Delete `coreclr.dll`, `mscorlib.dll` and `mscorlib.ni.dll` from that directory
-5. Copy all files in the coreclr bin directory to the test directory
-6. Run an OpenCover command with `xunit.console.netcore.exe`. For example:
+4. Delete `coreclr.dll`, `mscorlib.dll`, `mscorlib.ni.dll`, `System.Private.CoreLib.dll` and `System.Private.CoreLib.ni.dll` from that directory
```
Comment (stephentoub) [inline Documentation/building/code-coverage.md]: Is this step necessary, or will step 5 handle this when these files are overwritten?  Maybe we only need to list the .ni.dll files here, if they're not in the set of files that'll be built by default and copied over?

File: Documentation/building/facade-code-coverage.bat [hunk 2]
```diff
@@ -30,21 +30,36 @@ SET originalfolder=%cd%
 SET sourcefolder=%corefx%\src\%project%\tests
 
 SET coreclrbuild=%coreclr%\bin\Product\Windows_NT.x64.Debug
+SET coreclrbuild=%coreclr%\bin\Product\Windows_NT.x64.Release
```
Comment (stephentoub) [inline Documentation/building/facade-code-coverage.bat]: Setting the same variable twice?

File: Documentation/building/facade-code-coverage.bat [hunk 3]
```diff
@@ -30,21 +30,36 @@ SET originalfolder=%cd%
...
+del %targetdir%\CoreRun.exe
+del %targetdir%\CoreConsole.exe
+del %targetdir%\clretwrc.dll
+del %targetdir%\clrjit.dll
+del %targetdir%\dbgshim.dll
+del %targetdir%\mscordaccore.dll
+del %targetdir%\mscordbi.dll
+del %targetdir%\mscorrc.debug.dll
+del %targetdir%\mscorrc.dll
+del %targetdir%\sos.dll
```
Comment (stephentoub) [inline Documentation/building/facade-code-coverage.bat]: Related question; do all of these need to be deleted, or can we just delete the ones (if any) that won't be overwritten by the copy?

**Comment by stephentoub** [comment]
Thanks, @hughbe. Some questions/comments, otherwise LGTM.  Thanks for updating it.

File: Documentation/building/code-coverage.md (hunk 1 above)
Comment (hughbe) [inline Documentation/building/code-coverage.md]: I thought this at first, but this led me to Issue #8381

TLDR: to save space, we hard link the runtime files (coreclr.dll etc.). Thus, copying and overwriting files updates the central copy meaning that every single project is affected and uses the new copy.

File: Documentation/building/facade-code-coverage.bat (hunk 3 above)
Comment (hughbe) [inline Documentation/building/facade-code-coverage.bat]: See above response

File: Documentation/building/facade-code-coverage.bat (hunk 2 above)
Comment (hughbe) [inline Documentation/building/facade-code-coverage.bat]: Whoops. Fixed, thanks. Part of my ever on going battle with git to get it to do what I want

File: Documentation/building/code-coverage.md (hunk 1 above)
Comment (stephentoub) [inline Documentation/building/code-coverage.md]: Hmm. It's strange to me that an overwrite would be treated as a modification rather than as a deletion+creation. But I trust Matt's explanation. Thanks.
